server/src/pages/shot/view.js

```diff
@@ -212,7 +212,7 @@ class Body extends React.Component {
         <div>
           <div className="spacer"/>
           If you do nothing,<br/>
-          this shot will be permanently deleted <TimeDiff date={deleteTime} />.
+          this shot will be permanently deleted in <TimeDiff date={deleteTime} />.
```

Comment:
Sorry, I'm not very smart, but my eyes are telling me this is going to say something like:

> _"If you do nothing, this shot will be permanantly deleted in very soon."_

I think all other cases it will read fine though.